### ISSUE TITLE
Can not load the saved model with latent=CentBeta13

### ISSUE THREAD

**Issue by MySlientWind** [jfcrenshaw/pzflow#89]
When try to load the saved model with `latent=CentBeta13`, I get
```python
AttributeError: module 'pzflow.distributions' has no attribute 'CentBeta22'
```

Seems due to the CentBeta13 are saved with name "CentBeta22" thus can not be reloaded correctly?
https://github.com/jfcrenshaw/pzflow/blob/8e7a03fa857bfb3d0f6ce374d2e06204704dc13e/pzflow/distributions.py#L160

**Comment by jfcrenshaw** [comment]
Oops! I will fix this now and push an update.

**Comment by jfcrenshaw** [comment]
Okay, this can now be fixed by upgrading to v3.1.1!
